### PR TITLE
Using a string of eth as the only format for all kinds of prices and balances

### PR DESCRIPTION
**Pull request by julien51**
We are now using eth as the way to represent prices and balances. We use Eth as a string for now.
All conversions to wei are done *inside* the web3 provider.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
Fixes https://github.com/unlock-protocol/unlock/issues/1066

**Comment by julien51** [comment]
This also makes the `unit` attribute irrelevant... (and we were actually not using it anywhere but tests).

File: unlock-app/src/propTypes.js [hunk 1]
```diff
@@ -6,17 +6,18 @@ export const address = PropTypes.string
...
 export const lock = PropTypes.shape({
   address,
   name: PropTypes.string,
   expirationDuration: PropTypes.number,
-  keyPrice: PropTypes.string,
+  keyPrice: PropTypes.string, // Must be expressed in Eth!
   maxNumberOfKeys: PropTypes.number,
   owner: PropTypes.string,
   outstandingKeys: PropTypes.number,
+  balance: PropTypes.string, // Must be expressed in Eth!
```
Comment (cellog) [inline unlock-app/src/propTypes.js]: this adds a field to the lock propType. I don't have an issue with that, but is it doing 2 things in this PR?

Comment (julien51) [reply unlock-app/src/propTypes.js]: Well, I think this is still about the type/format of balances and prices?

**Pull request merged**